bio/webapp/src/org/intermine/bio/web/displayer/CytoscapeNetworkDisplayer.java

```diff
@@ -60,107 +37,5 @@ public CytoscapeNetworkDisplayer(ReportDisplayerConfig config, InterMineAPI im)
     @Override
     public void display(HttpServletRequest request, ReportObject reportObject) {
 
-        ObjectStore os = im.getObjectStore(); // Get OS
-        Model model = im.getModel(); // Get Model
-        Profile profile = SessionMethods.getProfile(request.getSession()); // Get Profile
-        PathQueryExecutor executor = im.getPathQueryExecutor(profile); // Get PathQueryExecutor
-
-        Set<Integer> startingFeatureSet = new LinkedHashSet<Integer>(); // feature: gene or protein
-        String featureType = "";
-
-        //=== Get Interaction information ===
-        Map<String, Set<String>> interactionInfoMap;
-        try {
-            interactionInfoMap = CytoscapeNetworkUtil.getInteractionInfo(model, executor);
-        } catch (ObjectStoreException e) {
-            throw new RuntimeException(e);
-        }
-
-        if (interactionInfoMap == null) {
-            request.setAttribute("dataNotIncludedMessage", DATA_NOT_INTEGRATED);
-            return;
-        }
-
-        //=== Handle object ===
-        // From gene report page
-        InterMineObject object = reportObject.getObject();
-        request.setAttribute("cytoscapeInteractionObjectId", object.getId());
```

Comment:
I think we'll need to keep lines 86 and 87 as well - merging this request in my local flymine results in a failed query. When I restored these two lines it went back to working ok. In short, the ID is needed to fetch the interaction data.